CRM/Contract/Form/EditMembership.php

```diff
@@ -134,18 +137,31 @@ public function postProcess () {
     $membership = $this->get('membership');
     $submitted = $this->exportValues();
 
-    Api4\Membership::update(FALSE)
+    $membership_update = Api4\Membership::update(FALSE)
       ->addWhere('id', '=', $membership['id'])
       ->addValue('join_date',                               $submitted['join_date']           )
-      ->addValue('membership_general.contract_file',        $submitted['contract_file']       )
       ->addValue('membership_general.membership_channel',   $submitted['membership_channel']  )
       ->addValue('membership_general.membership_contract',  $submitted['membership_contract'] )
       ->addValue('membership_general.membership_dialoger',  $submitted['membership_dialoger'] )
       ->addValue('membership_general.membership_reference', $submitted['membership_reference'])
       ->addValue('membership_referral.membership_referrer', $submitted['membership_referrer'] )
       ->addValue('start_date',                              $submitted['start_date']          )
-      ->addValue('status_id',                               $membership['status_id']          )
-      ->execute();
+      ->addValue('status_id',                               $membership['status_id']          );
+
+    $contract_file_metadata = $this->getElement('contract_file')->getValue();
+
+    if (!empty($contract_file_metadata)) {
+      $contract_file = Api4\File::create(FALSE)
+        ->addValue('content',   file_get_contents($contract_file_metadata['tmp_name']))
```

Comment:
Unfortunately, Civi 5.69 doesn't support `content` yet. 🥹 

You can probably do it similarly to this (i.e. create the file yourself, and then set `uri` to the filename instead of setting `content`):
https://github.com/greenpeace-cee/at.greenpeace.donutapp/blob/e627f70e9fa06131970e88b573d2d44265f2d0fd/CRM/Donutapp/Processor/Greenpeace/Donation.php#L289-L296